src/auth-service/utils/control-access.js

```diff
@@ -34,96 +33,6 @@ const generateClientSecret = (length) => {
   return clientSecret;
 };
 
-const NetworkModel = (tenant) => {
-  try {
-    const networks = mongoose.model("networks");
-    return networks;
-  } catch (error) {
-    const networks = getModelByTenant(tenant, "network", NetworkSchema);
-    return networks;
-  }
-};
-
-const UserModel = (tenant) => {
-  try {
-    let users = mongoose.model("users");
-    return users;
-  } catch (error) {
-    let users = getModelByTenant(tenant, "user", UserSchema);
-    return users;
-  }
-};
-
-const AccessTokenModel = (tenant) => {
-  try {
-    let tokens = mongoose.model("access_tokens");
-    return tokens;
-  } catch (error) {
-    let tokens = getModelByTenant(tenant, "access_token", AccessTokenSchema);
-    return tokens;
-  }
-};
-
-const PermissionModel = (tenant) => {
```

Comment:
@BenjaminSsempala , because of this refactor, then we just import the models directly and pass in the tenant parameter. No need for such now.

Reply:
Thanks @Baalmart